### PR TITLE
Add resources to PHP and JavaScript in Danish

### PR DESCRIPTION
**Pull request by Aeka123**
## What does this PR do?
Add resources to PHP and JavaScript

### Description
Necessary resources for beginners to learn JavaScript and PHP in Danish.
And I've removed unused classes in the C++ and C# sections.

### Why is this valuable (or not)?
It expands the variety of resources to include other programming languages as well. Which are both industry standards.

### How do we know it's really free?
There's no pay-wall on any of the resources I've found, and there is nothing on the individual websites to prove otherwise.

### For book lists, is it a book? For course lists, is it a course? etc.
My PR includes 1 book and 5 HTML pages with important beginner resources.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by eshellman — CHANGES_REQUESTED** [review]

File: books/free-programming-books-da.md [hunk 1]
```diff
@@ -13,12 +15,12 @@
...
 
 
-### <a id="csharp"></a>C\#
+### C\#
 
 * [Object-oriented Programming in C#](http://people.cs.aau.dk/~normark/oop-csharp/pdf/all.pdf) - Kurt Nørmark (PDF)
 
 
-### <a id="cpp"></a>C++
+### C++
```
[the remark this answers is not in the document]
Comment (Aeka123) [reply books/free-programming-books-da.md:23]: Huh... must've missed something... I'll fix it

**Comment by Aeka123** [comment]
@eshellman would you mind having a look at the latest changes? :-)

File: books/free-programming-books-da.md [hunk 2]
```diff
@@ -28,6 +30,20 @@
 * [Objektorienteret programmering i Java](http://javabog.dk) - Jacob Nordfalk
 
 
+### JavaScript
+
+* [Introduktion til objektorienteret JavaScript](https://ecommerce-platforms.com/da/articles/introduction-to-object-oriented-javascript) - Bogdan Rancea
+* [JavaScript Promises - Ryd op i de asynkrone kald](http://qed.dk/poul-foged/2014/02/21/javascript-promises-ryd-op-de-asynkrone-kald/) - Poul Foged
```
Comment (eshellman) [inline books/free-programming-books-da.md:36]: this is a blog post, not a book. remove it

File: books/free-programming-books-da.md [hunk 3]
```diff
@@ -28,6 +30,20 @@
...
 
 * [Programmering i Pascal](http://people.cs.aau.dk/~normark/all-basis-97.pdf) - Kurt Nørmark (PDF)
+
+
+### PHP
+
+* [Klasser I PHP](http://halmark.dk/php/classes-in-php.html) - Halmark's Artikler
+* [Løkker/loops i PHP](https://www.phpartikler.dk/artikler/loekker.php) - PHP Artikler
+* [PHP if, then, else og switch](https://www.phpartikler.dk/artikler/begynder2.php) - PHP Artikler
+* [PHP og MYSQL](http://samples.pubhub.dk/9788778531636.pdf) - Kristian Langborg-Hansen (PDF)
```
Comment (eshellman) [inline books/free-programming-books-da.md:49]: this is an 18 page sample of a book.

**Review by eshellman — COMMENTED** [review]
none of these are books.

**Pull request closed without merging**